### PR TITLE
Unify save-buttons in admin-ui

### LINKED ISSUE #30118: Admin UI - Fixed save buttons on the bottom at the page
### Description

Currently the save-buttons of different pages are not consistent, sometimes you need to scroll down the whole page to access them (e.g. Edit Realm Settings) and sometimes they are fixed on the bottom of the page (e.g. Edit Client). 

editRealmSettings
editClient

IMO the fixed-buttons approach is much more usable so I tried to unify the buttons in this way. 

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andreas-blaettlinger**
Closes #30118.

Fixed save-buttons to the bottom of the page.

### CODE REVIEW

File: js/apps/admin-ui/src/user/UserForm.tsx [hunk 1]
```diff
@@ -327,37 +334,13 @@ export const UserForm = ({
...
-            data-testid={!user?.id ? "create-user" : "save-user"}
-            isDisabled={
-              !user?.id &&
-              !watchUsernameInput &&
-              realm.registrationEmailAsUsername === false
-            }
-            allowNonDirty
-            allowInvalid
-          >
-            {user?.id ? t("save") : t("create")}
```
Comment (edewit) [inline js/apps/admin-ui/src/user/UserForm.tsx:343]: here the button will say save or create, this is not the case anymore when you change it to the `FixedButtonsGroup`

Comment (andreas-blaettlinger) [reply js/apps/admin-ui/src/user/UserForm.tsx:343]: Introduced optional props for naming the buttons in FixedButtonGroup with the latest commit.

**Review by edewit — APPROVED** [review]

**Review by agagancarczyk — APPROVED** [review]

**Comment by agagancarczyk** [comment]
@andreas-blaettlinger The changes look good, but there are some tests failing https://github.com/keycloak/keycloak/actions/runs/9495585401/job/26174300197?pr=30119. Could you please review and fix the failing tests? Thx

**Comment by andreas-blaettlinger** [comment]
@agagancarczyk mh I think these are flaky or not related to my changes tbh. Locally it works just fine and my changes shouldn't affect these failing tests either.

Am I missing something?

**Pull request merged**